docs/launch-in-aws.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Launch CyberArk Conjur In The Cloud
+layout: page
+section: get-started
+---
+
+You can easily launch a private, production-ready Conjur server in the
+cloud using Conjur's official template for Amazon CloudFormation.
+
+## Prerequisites
+
+First, you'll need an Amazon Web Services account. If you already have
+one, skip ahead. If not, you can [create a free account
+here][aws-signup].
+
+Then you need to [download the Conjur CloudFormation
+template][cf-template]. It's a plain text file that instructs AWS how
+to create the Conjur stack. You'll need this for the next step.
+
+## Prepare to launch
+
+1. Sign-in to your Amazon Web Services dashboard and notice the search
+   bar directly under the "AWS Services" heaer. Search for
+   "CloudFormation" and hit <kdb>Enter</kbd> to navigate.
```

Comment:
I'm guessing kdb is a typo, unless you were looking for a tag to represent a kernel debugger? :)

Reply:
Fixed ;)